tests/acceptance/features/apiFullTextSearch/search.feature

```diff
@@ -83,3 +84,19 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search version restored files by content
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "test.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "test.txt"
+    And user "Alice" has restored version index "1" of file "test.txt"
+    When user "Alice" searches for "Content:hello" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result of user "Alice" should contain only these files:
+      | test.txt |
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
```

Comment:
here also if we can check the content of the file which has been restored.

Reply:
Okay :blush: